### ISSUE TITLE
Request for Standards position: policy container

### ISSUE THREAD

**Issue by mozfreddyb** [mozilla/standards-positions#863]
## Request for Mozilla Position on an Emerging Web Specification

* Specification Title: ["policy container" in HTML](https://html.spec.whatwg.org/#policy-containers)
* Specification or proposal URL (if available): [pull request that established it](https://github.com/whatwg/html/pull/6504)
* Explainer URL (if available): https://github.com/antosart/policy-container-explained
* Caniuse.com URL (optional): AFAIU this is not web exposed
* Bugzilla URL (optional): https://bugzilla.mozilla.org/show_bug.cgi?id=1706026
* Mozillians who can provide input (optional): @zcorpan

**Comment by zcorpan** [comment]
If it's not web exposed, it doesn't need a standards position. Maybe there are some details that are exposed though, like the effects of inheritance. But I still think any such differences were spec bugs to begin with.

OK to close without a position?

**Comment by mozfreddyb** [comment]
I don't think this needs to be on the dashboard, but I wanted to make sure there are no objections. I take that to be true given the lack of opposition here.

**Comment by mozfreddyb** [comment]
I need to take that back. There are some web-visible changes that should come from implementing this.

 In particular, there are some inheritance inconsistencies between browsers as highlighted by test cases in https://wpt.fyi/results/referrer-policy/generic/inheritance/iframe-inheritance-history-about-srcdoc.html?label=experimental&label=master&aligned

That being said, I think this is more of an argument in favor of doing this rather than anything else.

@zcorpan can you please confirm for posterity?
